tools/ansible/roles/dlpx.appliance-build-bootstrap/tasks/main.yml

```diff
@@ -0,0 +1,9 @@
+#
+# Copyright (c) 2018 by Delphix. All rights reserved.
+#
+
+#
+# This role configures a VM for building the Virtualization Engine.
+#
+---
+- include: install_zfsutils.yml
```

Comment:
IMO, we should just remove the `install_zfsutils.yml` file and put those contents directly in this file.. I don't see a need for two files.. but I'm OK with this.